test/test_datasets.py

```diff
@@ -1,18 +1,18 @@
 import contextlib
-import sys
 import os
 import unittest
 from unittest import mock
 import numpy as np
 import PIL
 from PIL import Image
-from torch._utils_internal import get_file_path_2
 import torchvision
 from torchvision.datasets import utils
-from common_utils import get_tmp_dir
-from fakedata_generation import svhn_root, places365_root, widerface_root, stl10_root
+from fakedata_generation import (
+    places365_root,
+    stl10_root,
+    svhn_root,
+)
 import xml.etree.ElementTree as ET
-from urllib.request import Request, urlopen
```

Comment:
Same as above. I know the imports are messy right now, but fixing this does not belong in this PR. I suggest we wait with this until #3531 is closed.